docs/versioned_docs/version-4.1.x/releases.md

```diff
@@ -58,12 +72,36 @@ Grafana Dashboards for Replicated PV Mayastor has been added in this releases.
 
 ### Fixed Issues - Local Storage
 
+- **Environment Variable**
+
+Ensures that the environment variable to disable event analytics reporting is properly honored. ([#211](https://github.com/openebs/dynamic-localpv-provisioner/pull/211))
+
+- **Prevent Race Condition**
+
+Adds `priorityClassName` to pods to prevent race conditions caused by pod eviction. ([#182](https://github.com/openebs/dynamic-localpv-provisioner/pull/182))
```

Comment:
How does this prevent race conditions? @Abhinandan-Purkait @niladrih

Reply:
priorityClassName sets the urgency in scheduling order for Pods. Pods, by default have a default, and equal priority when being scheduled on to a node. However, if a higher priority is assigned using a custom/special priorityClass, then critic components (like our storage controllers) have a higher priority when being picked up by a scheduler.

Reply:
No it doesn't prevent any race condition. It just helps in scheduling.

Reply:
Yeah I guess we can remove it altogether